diavlos/src/diaugeia/diaugeia.py

```diff
@@ -24,10 +25,10 @@ def fetch(self, code):
                     enum, if an ada for the code is not found.
                 dict: ada information
         """
-        resource = f'{API_TEST_URL}{code}.json'
+        resource = f'{API_BASE_URL}{code}.json'
         try:
             response = requests.get(resource).json()
-        except ConnectionError:
+        except ConnectionError or JSONDecodeError:
```

Comment:
I don't know if this is correct. If it is correct, it's most likely not standard. [Better do](https://docs.python.org/3/tutorial/errors.html#handling-exceptions): 
```python
except (ConnectionError, JSONDecodeError):
```

Reply:
> I don't know if this is correct

Turns out it is **not** correct, since
```python
ConnectionError or JSONDecodeError
```
evaluates to:
```python
ConnectionError
```

So, the correction you made is basically still equivalent to:
```python
exception ConnectionError:
```

Meaning that if `JSONDecodeError` is raised, it won't be caught.

```python
except (ConnectionError, JSONDecodeError):
```

is what you want.